pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -34,28 +34,8 @@ func NewResourceSyncController(
 		return nil, err
 	}
 	if err := resourceSyncController.SyncConfigMap(
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespaceName, Name: "aggregator-client-ca"},
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.MachineSpecifiedGlobalConfigNamespace, Name: "aggregator-client-ca"},
-	); err != nil {
-		return nil, err
-	}
-	if err := resourceSyncController.SyncConfigMap(
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.OperatorNamespace, Name: "initial-client-ca"},
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.UserSpecifiedGlobalConfigNamespace, Name: "initial-client-ca"},
```

Comment:
where is that gone?

Reply:
bad copy paste that I guess I didn't need later :(